diskfs_other.go

```diff
@@ -0,0 +1,19 @@
+//go:build !windows && !linux && !darwin
+// +build !windows,!linux,!darwin
```

Comment:
The `go.mod` now has go 1.19; I don't think you need the obsolete `// +build` line anymore

Reply:
Removed. Thank.